### PR TITLE
Update helloworld sample instructions

### PR DESCRIPTION
**Pull request by frankbu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by geeknoid** [comment]
Rebase in order to fix CI problem.

File: samples/websockets/route.yaml [hunk 1]
```diff
@@ -30,4 +30,3 @@ spec:
     - destination:
         host: tornado
       weight: 100
-    websocketUpgrade: true
```
Comment (CameronNemo) [inline samples/websockets/route.yaml:33]: Please explain why this seemingly unrelated change was included in this PR.

Comment (CameronNemo) [reply samples/websockets/route.yaml:33]: @frankbu @geeknoid

Comment (frankbu) [reply samples/websockets/route.yaml:33]: It's a while ago, so I can't remember. It looks like it might have been accidentally added to the wrong branch, but it does look correct to remove it, since that field is not documented or supported.
